pkgs/games/legendary-gl/default.nix

```diff
@@ -7,23 +7,27 @@
 
 buildPythonApplication rec {
   pname = "legendary-gl"; # Name in pypi
-  version = "0.20.1";
+  version = "0.20.6";
 
   src = fetchFromGitHub {
     owner = "derrod";
     repo = "legendary";
     rev = version;
-    sha256 = "0kdrhdvh9gxq1zinh5w852f5fjls9902pcrkhkhm2c0vvq7jfass";
+    sha256 = "1v6jbnasz2ilcafs6qyl6na4a8cxy2lgwr0hqsja6d846rfqa8ad";
   };
 
   propagatedBuildInputs = [ requests ];
 
   disabled = pythonOlder "3.8";
 
+  doCheck = false;
```

Comment:
Even though legendary-gl does not define any tests, I would still prefer to leave doCheck as-is. For now, it doesn't make a difference, and if they ever add tests in the future, we're already in the right place.

Otherwise LGTM!